### PR TITLE
feat: UI improvements and fixes

### PR DESCRIPTION
**Pull request by guansss**
- fix(account): activation button ui @martinwang2002 
- feat: thicker scrollbar on windows (#76) @martinwang2002 
- feat: add backToTop button @FutabaRio 
- fix: 在 unmount 时移除监听 @FutabaRio 
- fix: error caused by level cache with an empty array
- fix: missing React keys
- fix: requests may be sent before token is initialized
- feat: operation batch upload
batch

### CODE REVIEW

File: src/components/BackToTop.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  const [showButton, setShowButton] = useState(false)
+  useEffect(() => {
+    const handleScroll = () => {
+      if (window.pageYOffset > 100) {
+        setShowButton(true)
+      } else {
+        setShowButton(false)
+      }
+    }
+    document.addEventListener('scroll', handleScroll)
```
Comment (GalvinGao) [inline src/components/BackToTop.tsx:15]:
```suggestion
    document.addEventListener('scroll', handleScroll, { passive: true })
```

See https://developer.chrome.com/docs/lighthouse/best-practices/uses-passive-event-listeners/

File: src/components/BackToTop.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    return () => {
+      document.removeEventListener('scroll', handleScroll)
```
Comment (GalvinGao) [inline src/components/BackToTop.tsx:17]:
```suggestion
      document.removeEventListener('scroll', handleScroll, { passive: true })
```

File: src/components/Effects.tsx [hunk 3]
```diff
@@ -66,6 +72,9 @@ export const Effects: FC = () => {
           }
 
           if (shouldLogout) {
+            // setting the state synchronously may cause problems (not really sure), so we add a small delay
+            await new Promise((resolve) => setTimeout(resolve, 50))
```
Comment (GalvinGao) [inline src/components/Effects.tsx:76]: A bit hacky. Add a `FIXME` for clarification & notice for future maintainers.

```suggestion
            // FIXME: setting the state synchronously may cause problems (not really sure), so we add a small delay for a temporary hack
            await new Promise((resolve) => setTimeout(resolve, 50))
```

**Review by GalvinGao — CHANGES_REQUESTED** [review]
Some minor tweaks might need to be addressed. Other parts LGTM

File: src/components/BackToTop.tsx (hunk 1 above)
Comment (guansss) [reply src/components/BackToTop.tsx:15]: Only mouse and touch events need this, the scroll event cannot be default-prevented.

> Lighthouse uses the following process to identify event listeners that may affect scrolling performance:
> 1. Collect all event listeners on the page.
> 2. Filter out non-touch and non-wheel listeners.
> ...

**Review by GalvinGao — APPROVED** [review]
LGTM! お疲れ様でした！

**Pull request merged**